### PR TITLE
refactor: use whiskers

### PR DESCRIPTION
**Pull request by uncenter**
Adds [Whiskers](https://github.com/catppuccin/toolbox/tree/main/whiskers), Catppuccin's internal templating tool, to build the themes. Instead of editing the `.yaml` files directly, edit the `templates/glazewm.tera` file and then run `just build` to build the output themes and example configuration. You will need Whiskers (linked above) and *optionally* [Just](https://github.com/casey/just) installed.

### CODE REVIEW

**Comment by fathulfahmy** [comment]
will test it by the end of the weekend and merge once passed

File: justfile [hunk 1]
```diff
@@ -1,6 +1,11 @@
+# use powershell on windows
+set windows-shell := ["powershell.exe", "-NoLogo", "-Command"]
+
+# list available recipes by default
 _default:
   @just --list
 
+# build all flavors and example config files
```
Comment (uncenter) [inline justfile:8]: Are these comments needed? Seems pretty self explanatory to me but I guess.

Comment (fathulfahmy) [reply justfile:8]: Wouldn't it provide explanation of each function at a glance without needing to go through the Just documentation?

Comment (uncenter) [reply justfile:8]: Maybe its just me but I see "default" next to "list" and I can tell that the default tasks will list available tasks. And the "build" task pretty clearly builds the themes :)

Comment (fathulfahmy) [reply justfile:8]: Made a new commit removing the comments since the syntax is very readable. Thank you for your work!

**Pull request merged**